TB-Console/Services/TagService.cs

```diff
@@ -50,13 +50,31 @@ public Task<LocationTagResponse> AddLocationTags(LocationTagRequest request)
                     LocationTags = locationTags
                 };
 
+            return Task.FromResult(response);
```

Comment:
We should get rid of this Task.FromResult call. Lets try making the method async, then await the AddRangeAsync, then just return the response. Lets see if that works.

Reply:
and save changes async too I think